apis/env.d.ts

```diff
@@ -14,6 +14,12 @@ export const env: {
     [key: string]: any,
   },
   profiles: string[],
+  log: {
+    user: boolean,
+    levels: Record<string, 'silent' | 'info' | 'warn' | 'error' | 'debug' | 'trace' | 'silly' | 'verbose'>,
```

Comment:
```suggestion
    levels: Record<string, Lowercase<keyof typeof levels>>,
```

if you want to include this, this change also requires

```ts
import { levels } from './log'
```